<?xml version="1.0"?>
<!-- Launches the ROS-based simulation. -->

<launch>
  <arg name="config"/>

  <!-- use simulation time -->
  <param name="/use_sim_time" value="true" type="bool"/>

  <!-- Simulation -->
  <node pkg="mmseq_run" type="sim_ros.py" name="sim_ros" output="screen" required="true" args="--config $(arg config) --GUI" />

</launch>